Analysis/config/sources.cfg

```diff
@@ -142,6 +151,15 @@ cuts: LM -1
 training_range: SM 0:29
 samples: LM TT Signal_Radion Signal_Graviton
 
+[MVA mva_lmANkin_noparams]
+training: LM hh-bbtautau/Analysis/config/mva/new_mva_trainings/myFactoryBDT-LowMass_noparameters_kinvarsAN_train0-29_Grad_1.weights.xml
+variables: LM MT_l1 mass_H_kinfit costheta_METhtautau_sv phi_2_sv mass_H mass_H_kinfit_chi2 mass_H_vis dR_hbbMET MT2 MT_htautau_sv dphi_l1l2 p_zetavisible dphi_l1MET pt_hbb dR_l1l2Pt_htautau_sv dphi_b1b2 pt_htautau mass_top1 pt_MET dR_l1l2Pt_htautau
+masses: LM 250 260 270 280 300 320 250 260 270 280 300 320
+spins: LM    0   0   0   0   0   0   2   2   2   2   2   2
```

Comment:
It is non-parametric, so should be only one point (as for legacy). Same for non-param setups below.